scripts/exporter.js

```diff
@@ -95,7 +95,11 @@ function dedupeSentences(languageCode, sentences, path) {
   const numberOfDupes = sentences.length - dedupedSentences.length;
   console.log(`  - Got ${numberOfDupes} duplicated sentences in Sentence Collector..`);
 
-  const notAlreadyExistingInCV = dedupedSentences.filter((sentence) => !alreadyExistingCVSentences.includes(sentence));
+  const cvSentences = alreadyExistingCVSentences.reduce((acc, sentence) => {
+    acc[sentence] = true;
+    return acc;
+  }, {});
+  const notAlreadyExistingInCV = dedupedSentences.filter((sentence) => !cvSentences.hasOwnProperty(sentence));
```

Comment:
102:85  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins